lib.py

```diff
@@ -346,22 +346,22 @@ def pay(self, amount_msat: int, binding = None):
 
                     status = payment["status"]
                     if status != "complete":
-                        self._plugin.log(f"Failed to pay member {member_id}")
+                        self._plugin.log(f"Failed to pay member {m.id}")
                         continue
 
                     # update the member outlay with the payment amount (respecting fee accounting)
-                    total_amount_sent = payment["amount_sent_msat"]
-                    total_amount_sent_minus_fees = payment["amount_msat"]
+                    total_amount_sent: Millisatoshi = payment["amount_sent_msat"]
+                    total_amount_sent_minus_fees: Millisatoshi = payment["amount_msat"]
```

Comment:
I am just specifying the type here. It appears these fields are being cast to type Millisatoshi at some point